8.0/browsers/Dockerfile

```diff
@@ -11,9 +11,6 @@ RUN curl -sSL -o selenium-server-standalone-${SELENIUM_VER}.jar "https://seleniu
     rm selenium-server-standalone-${SELENIUM_VER}.jar
 
 RUN sudo apt-get update && \
-	sudo apt-get install --yes --no-install-recommends \
-		xvfb \
-	&& \
```

Comment:
You forgot to update your shared tools when doing this PR. This removal (and the other one) shouldn't happen.